website/src/components/CaasAvatar.tsx

```diff
@@ -1,20 +1,24 @@
+import Link from "@docusaurus/Link";
 import CaasLogo from "/img/CaaS-Logo.svg";
 
 export default function CaasAvatar(): JSX.Element {
   const name = "CaaS@DTIT";
-  const description = "The Container as a Service Team at Deutsch Telekom IT";
+  const description = "The container as a service team at Deutsch Telekom IT";
```

Comment:
```suggestion
  const description = "The container as a service team at Deutsche Telekom IT";
```

I know our name is CaaS@DTIT but it might be a consideration to write from the Deutsche Telekom IT. 🤷‍♂️